src/frontend/src/app/lobby/lobby.component.html

```diff
@@ -1,7 +1,27 @@
-<h2>Game ID: {{ id }}</h2>
+<button class="button-topleft" (click)="leave()">Leave</button>
 
-<div *ngFor="let player of players">
-  <img src="{{ player.picture }}" />
-  {{ player.name }}
+<div align="center">
+  <h1 class="title">CodeChamp</h1>
+
```

Comment:
Can you add some indicator saying this is the lobby (title/header)

Reply:
Maybe we should also add an indicator for the number of players in the lobby (e.g. 3/20)?

Reply:
![image](https://user-images.githubusercontent.com/33377586/211447966-8e286343-edc9-4c14-bf9b-12c7ad355768.png)